docs/EN_US/ECLStandardLibraryReference/SLR-Mods/SprayFixed.xml

```diff
@@ -183,6 +186,17 @@
           FALSE.</entry>
         </row>
 
+        <row>
+          <entry><emphasis>noCommon</emphasis></entry>
+
+          <entry>Optional. A boolean TRUE or FALSE flag for "commoning up" of
+          pull or push processes on same host. Set to FALSE to "common up"
+          pull or push processes on same host. Default can be set in
```

Comment:
I think it would be better to reword the second sentence, because I think there is nearly repeat the first one "pull or push processes on same host". Maybe "Set to FALSE to "common up" the operation [on [same] host]." ?